.golangci.yml

```diff
@@ -24,6 +24,8 @@ linters:
     - whitespace
     - loggercheck
     - sloglint
+    - exptostd
```

Comment:
please mention this change in the PR

Reply:
Thanks @machine424 
Done.

Reply:
Thanks @mmorel-35 
Sorted alphabetically done.

Reply:
> keep this list sorted alphabetically

Thanks. Got it.